spirv_msl.hpp

```diff
@@ -456,8 +481,12 @@ class CompilerMSL : public CompilerGLSL
 	void set_argument_buffer_device_address_space(uint32_t desc_set, bool device_storage);
 
 	// Query after compilation is done. This allows you to check if a location or set/binding combination was used by the shader.
+	// Deprecated: use is_msl_shader_input_used() instead.
```

Comment:
There is a SPIRV_CROSS_DEPRECATED macro for this.

Reply:
Done. I couldn't put it on the deprecated enums, because I don't think every compiler supports putting attributes on enumerants.